dockers/docker-dhcp-server/cli/show/plugins/show_dhcp_server.py

```diff
@@ -79,5 +79,24 @@ def range(db, range_name):
     click.echo(tabulate(table, headers=headers))
 
 
+@ipv4.command()
+@click.argument('interface', required=False)
+@clicommon.pass_db
+def port(db, interface):
+    if not interface:
+        interface = "*"
+    headers = ["Interface", "Bind"]
+    table = []
+    dbconn = db.db
+    for key in dbconn.keys("CONFIG_DB", "DHCP_SERVER_IPV4_PORT|*|" + interface):
```

Comment:
Suggest changing var `interface` to `dhcp_interface` and searching key `"DHCP_SERVER_IPV4_PORT|" + dhcp_interface + "|*"`. Maybe in future we will support single physical port DHCP, means that key would be like "DHCP_SERVER_IPV4_PORT|Ethernet5"